lib/GraphView.dart

```diff
@@ -59,26 +60,28 @@ class GraphView extends MultiChildRenderObjectWidget {
 }
 
 class RenderCustomLayoutBox extends RenderBox
-    with ContainerRenderObjectMixin<RenderBox, NodeBoxData>, RenderBoxContainerDefaultsMixin<RenderBox, NodeBoxData> {
-  Graph _graph;
-  Layout _algorithm;
-  Paint _paint;
+    with
+        ContainerRenderObjectMixin<RenderBox, NodeBoxData>,
+        RenderBoxContainerDefaultsMixin<RenderBox, NodeBoxData> {
+  Graph? _graph;
+  Layout? _algorithm;
+  Paint? _paint;
```

Comment:
❓ if the `Graph` is nonNullable in constructor, why it might be nullable as variable and why do you might need to set it to null?

Reply:
Same question about `Layout`